lambda/delete-user-services/tests/delete-user-services.test.ts

```diff
@@ -0,0 +1,91 @@
+import { DynamoDBDocumentClient, DeleteCommand } from "@aws-sdk/lib-dynamodb";
+import { SendMessageCommand, SQSClient } from "@aws-sdk/client-sqs";
+import { mockClient } from "aws-sdk-client-mock";
+import {
+  handler,
+  validateUserData,
+  deleteUserData,
+} from "../delete-user-services";
+
+import { TEST_USER_DATA, TEST_SNS_EVENT } from "./testHelpers";
+
+const dynamoMock = mockClient(DynamoDBDocumentClient);
+const sqsMock = mockClient(SQSClient);
+
+describe("deleteUserData", () => {
+  beforeEach(() => {
+    dynamoMock.reset();
+
+    process.env.TABLE_NAME = "TABLE_NAME";
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("deletes item from DynamoDB", async () => {
+    await deleteUserData(TEST_USER_DATA);
+    expect(dynamoMock.commandCalls(DeleteCommand).length).toEqual(1);
```

Comment:
Could you add another assertion that it call the delete command with the user ID? 
We could use the [Jest matchers](https://github.com/m-radzikowski/aws-sdk-client-mock#jest-matchers) like this (untested)

```suggestion
    expect(dynamoMock.commandCalls(DeleteCommand).length).toEqual(1);
    expect(dynamoMock).toHaveReceivedCommand(DeleteCommand, {
      TableName: process.env.TABLE_NAME,
      Key: { user_id: TEST_USER_DATA.user_id },
    });
```

Reply:
Ok so this solves the problem:
https://github.com/alphagov/di-account-management-backend/pull/38/commits/d8f8e704ddf40339d7e02c3df2eb5a681db895da

Turns out when I console.logged the env it _was_ there. So not a jest thing.

Seems to work fine on the lambda, so I wonder if there's something about the way Jest Invokes these... like not the file just the fucntion?

Bit odd.

Anyway there's no re-use around the file for these, so seems safe to put them directly into the function.